sql-ballerina/src/sql/types.bal

```diff
@@ -392,9 +392,316 @@ class ResultIterator {
     }
 }
 
-# Represents SQL OutParameter used in procedure calls.
-public class OutParameter {
+# Represents Char Out Parameter used in procedure calls
+public class CharOutParameter {
+    # Parses returned Char SQL value to ballerina value.
+    #
+    # + td - Type description of the data that need to be converted
+    # + return - The converted ballerina value or Error
+    public isolated function get(typedesc<anydata> td) returns td|Error = @java:Method {
```

Comment:
Shall we a more meaningful parameter name like ``typeDescription` or ``typeDesc` instead of `td` for all the occurrences?

Reply:
Fixed in b39da5c